cs-minibot-platform-src/src/main/resources/public/gui/index.html

```diff
@@ -107,9 +107,13 @@
 		<button id="addBot">add bot</button>
 		<button id = "updateLocs"> Run simulator </button>
 		<br><hr><br>
-		Scenario:
-		<textarea rows="4" cols="50" name="scenario" id="scenario">[{"type": "scenario_object", "position":"[5,5]", "angle":45, "size":1},{"type": "scenario_object", "position":"[3,3]", "angle":45, "size": 1}]</textarea>
+		Scenario name:<input type="text" id="scenarioname"
+							 value="scenario1"><br>
+		Scenario viewer: <textarea rows="4" cols="50" name="scenario"
+					   id="scenario">[{"type":	"scenario_object", "position":"[5,5]", "angle":45, "size":1},{"type": "scenario_object", "position":"[3,3]", "angle":45, "size": 1}]</textarea>
 		<button id="addScenario">add scenario</button>
+		<button id="loadScenario">load scenario</button>
```

Comment:
maybe rename these buttons to be extra clear to everyone else on the team- from what you showed me, I would see add scenario as load scenario from text box, load scenario as fetch scenario to text box, and save scenario as save scenario from textbox